vim/vscode-code-actions.vim

```diff
@@ -70,39 +64,39 @@ nnoremap <expr> = <SID>vscodeFormat()
+nnoremap <silent> gd <Cmd>call <SID>vscodeGoToDefinition("Definition")<CR>
+nnoremap <silent> <C-]> <Cmd>call <SID>vscodeGoToDefinition("Definition")<CR>
+nnoremap <silent> gO <Cmd>call <SID>vscodeNotifyWithMark('workbench.action.gotoSymbol')<CR>
+nnoremap <silent> gF <Cmd>call VSCodeNotify('editor.action.peekDeclaration')<CR>
+nnoremap <silent> gD <Cmd>call VSCodeNotify('editor.action.peekDefinition')<CR>
+nnoremap <silent> gH <Cmd>call VSCodeNotify('editor.action.referenceSearch.trigger')<CR>
+
+xnoremap <silent> K <Cmd>call VSCodeNotify('editor.action.showHover')<CR>
+xnoremap <silent> gh <Cmd>call VSCodeNotify('editor.action.showHover')<CR>
+xnoremap <silent> gf <Cmd>call <SID>vscodeGoToDefinition("Declaration")<CR>
+xnoremap <silent> gd <Cmd>call <SID>vscodeGoToDefinition("Definition")<CR>
+xnoremap <silent> <C-]> <Cmd>call <SID>vscodeGoToDefinition("Definition")<CR>
+xnoremap <silent> gO <Cmd>call <SID>vscodeNotifyWithMark('workbench.action.gotoSymbol')<CR>
+xnoremap <silent> gF <Cmd>call VSCodeNotify('editor.action.peekDeclaration')<CR>
+xnoremap <silent> gD <Cmd>call VSCodeNotify('editor.action.peekDefinition')<CR>
+xnoremap <silent> gH <Cmd>call VSCodeNotify('editor.action.referenceSearch.trigger')<CR>
 
 " <C-w> gf opens definition on the side
-nnoremap <silent> <C-w>gf :<C-u>call VSCodeNotify('editor.action.revealDeclarationAside')<CR>
-nnoremap <silent> <C-w>gd :<C-u>call VSCodeNotify('editor.action.revealDefinitionAside')<CR>
-xnoremap <silent> <C-w>gf :<C-u>call VSCodeNotify('editor.action.revealDeclarationAside')<CR>
-xnoremap <silent> <C-w>gd :<C-u>call VSCodeNotify('editor.action.revealDefinitionAside')<CR>
+nnoremap <silent> <C-w>gf <Cmd>call VSCodeNotify('editor.action.revealDeclarationAside')<CR>
+nnoremap <silent> <C-w>gd <Cmd>call VSCodeNotify('editor.action.revealDefinitionAside')<CR>
+xnoremap <silent> <C-w>gf <Cmd>call VSCodeNotify('editor.action.revealDeclarationAside')<CR>
+xnoremap <silent> <C-w>gd <Cmd>call VSCodeNotify('editor.action.revealDefinitionAside')<CR>
 
 " Bind C-/ to vscode commentary since calling from vscode produces double comments due to multiple cursors
 xnoremap <expr> <C-/> <SID>vscodeCommentary()
 nnoremap <expr> <C-/> <SID>vscodeCommentary() . '_'
 
 " Workaround for gk/gj
-nnoremap gk :<C-u>call VSCodeCall('cursorMove', { 'to': 'up', 'by': 'wrappedLine', 'value': v:count ? v:count : 1 })<CR>
-nnoremap gj :<C-u>call VSCodeCall('cursorMove', { 'to': 'down', 'by': 'wrappedLine', 'value': v:count ? v:count : 1 })<CR>
+nnoremap gk <Cmd>call VSCodeCall('cursorMove', { 'to': 'up', 'by': 'wrappedLine', 'value': v:count ? v:count : 1 })<CR>
+nnoremap gj <Cmd>call VSCodeCall('cursorMove', { 'to': 'down', 'by': 'wrappedLine', 'value': v:count ? v:count : 1 })<CR>
 
 " workaround for calling command picker in visual mode
-xnoremap <silent> <C-P> :<C-u>call <SID>openVSCodeCommandsInVisualMode()<CR>
+xnoremap <silent> <C-P> <Cmd>call <SID>openVSCodeCommandsInVisualMode()<CR>
```

Comment:
Now this command fail tests.

Reply:
I have no idea what causes it. Either there is something wrong with the test or it just randomly failed due to timing changes. Maybe putting `wait()` commands in the test will fix it.

Reply:
Yes, I don't know either. The command works correctly for me.

Reply:
I found solution! The problem was in `visualmode()`.